### DISCUSSION TITLE
CMultiSelect Keyboard navigation and options selection

### DISCUSSION THREAD

**Discussion by saba-softiez** [coreui/coreui-react#354] [Q&A]
Hi All,
Is there a way to use keyboard navigation for selecting CMultiselect options?

Thank you,
Saba

**Comment by mrholek** [comment]
@saba-softiez when CMultiSelect is open, you can use the tab to move to the options list, then you can use tab, shift+ tab, or keyboard. To select an item please use space or enter.
